HaruhiGekidouCLI/ArcCommand.cs

```diff
@@ -72,17 +72,22 @@ public override int Invoke(IEnumerable<string> arguments)
         else if (_pack)
         {
             GekidouArc arc = new();
-            string[] directories = Directory.GetDirectories(_input, "*", SearchOption.AllDirectories).Order().ToArray();
+            List<string> directories = [];
+            directories.Add(_input);
+            directories.AddRange(Directory.GetDirectories(_input, "*", SearchOption.AllDirectories).Order().ToArray());
+            
             arc.Entries.Add(new(string.Empty, true, depth: 0));
             foreach (string dir in directories)
             {
                 int actualDepth = Path.GetRelativePath(_input, dir).Split(Path.DirectorySeparatorChar).Length;
```

Comment:
I'd remove the if statement below and instead write this as:
```suggestion
                int actualDepth = dir.Equals(_input) ? 1 : Path.GetRelativePath(_input, dir).Split(Path.DirectorySeparatorChar).Length + 1;
```